components/ResourceCreation/TestResourcesDisplay.tsx

```diff
@@ -5,43 +5,101 @@ import { patientTestCaseState } from '../../state/atoms/patientTestCase';
 import { valueSetMapState } from '../../state/selectors/valueSetsMap';
 import { dataRequirementsState } from '../../state/selectors/dataRequirements';
 import { selectedDataRequirementState } from '../../state/atoms/selectedDataRequirement';
+import Fuse from 'fuse.js';
+import SearchBar from './SearchBar';
+import { searchQueryState } from '../../state/atoms/searchQuery';
+import { useEffect, useState } from 'react';
+import { DataRequirement } from 'fhir/r4';
+
+interface DataSearchKeys {
+  type: string;
+  id: string | undefined;
+  displayString: string;
+}
 
 export default function TestResourcesDisplay() {
   const dataRequirements = useRecoilValue(dataRequirementsState);
   const [, setSelectedDataRequirement] = useRecoilState(selectedDataRequirementState);
   const valueSetMap = useRecoilValue(valueSetMapState);
   const currentPatients = useRecoilValue(patientTestCaseState);
+  const searchQuery = useRecoilValue(searchQueryState);
+  const [dataReqsToDisplay, setDataReqsToDisplay] = useState<DataRequirement[] | null>(dataRequirements);
 
-  return dataRequirements?.length && Object.keys(currentPatients).length ? (
-    <div
-      data-testid="test-resource-panel"
-      style={{
-        maxHeight: '40vh',
-        overflow: 'scroll'
-      }}
-    >
-      <Stack>
-        {dataRequirements.map((dr, i) => {
-          const key = `${dr.type}${i}`;
+  useEffect(() => {
+    /**
+     * Assembles an array of data that will be searched by FuseJS to narrow
+     * down the displayed data requirements in the test resources display.
+     * @returns array of data to use for fuzzy searching
+     */
+    const getSearchContent = (): DataSearchKeys[] | undefined => {
+      if (dataRequirements) {
+        const searchableData = dataRequirements.map(dr => {
+          const type = dr.type;
+          const id = dr.id;
```

Comment:
Yeah we don't seen to need this because as far as I can tell it's always undefined.

Reply:
Good catch, I forgot to remove this - I think I was going to try and use it somewhere else in the code for filtering the data requirements but we can just remove it since it never gets used for anything and isn't relevant for searching